### ISSUE TITLE
Add .sbs/.sbsar support/integration.

### ISSUE THREAD

**Issue by audiotrope** [Facepunch/sbox-issues#6183]
### For?

S&Box, Material Editor

### What can't you do?

Currently, I can't view a substance material in-engine without having to manually create a vmat. 

### How would you like it to work?

I would like to be able to drag and drop .sbs files into the project folder, and have the option to create a vmat from that .sbs file. Maybe have it generate automatically?

### What have you tried?

Current workflow is as follows: create material in substance designer -> export textures to material directory -> create vmat -> tune vmat to make appearance match that of substance.

### Additional context

In general, greater support and integration with popular tools and common file formats would be greatly appriciated.

**Comment by QuackCola** [comment]
I remember doing a request like this a few years ago. this is something that could be done via a library. https://github.com/Facepunch/sbox-issues/issues/1959

**Comment by audiotrope** [comment]
> I remember doing a request like this a few years ago. this is something that could be done via a library possibly by soemone in the community. #1959

Yeah, I read it, figured I'd give it another shot to see what would be said now.
